reporting_app/static/library_preps.js

```diff
@@ -0,0 +1,207 @@
+
+var chart, library_data;  // globals for the chart object and the library data to be plotted
+
+// pointer to the current metric being plotted
+var active_colour_metric;
+var colour_palette = Highcharts.getOptions().colors;
+
+// Artifact y coordinates, but plot them as x coordinates as per Lims UI
+var heatmap_x_coords = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'];
+
+metrics = {
+    'pc_q30': {name: '%Q30', path: ['bioinformatics_qc', 'aggregated', 'pc_q30']},
+    'pc_mapped_reads': {name: '% mapped reads', path: ['bioinformatics_qc', 'aggregated', 'pc_mapped_reads']},
+    'species': {name: 'Species', path: ['bioinformatics_qc', 'species_name'], type: 'category'},
+    'matching_species': {name: 'Matching species', path: ['bioinformatics_qc', 'aggregated', 'matching_species'], type: 'category'},
+    'het_hom_ratio': {name: 'Het/hom ratio', path: ['bioinformatics_qc', 'sample_contamination', 'het_hom_ratio']},
+    'freemix': {name: 'Freemix', path: ['bioinformatics_qc', 'sample_contamination', 'het_hom_ratio']},
```

Comment:
`freemix` is pointing to `het_hom_ratio`
```suggestion
    'freemix': {name: 'Freemix', path: ['bioinformatics_qc', 'sample_contamination', 'freemix']},
```